exercises/pascals-triangle/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include "pascals_triangle.h"
+
+static void zero_out(size_t ** input, size_t count)
+{
+   size_t i, j;
+   for (i = 0; i < count; i++) {
+      for (j = 0; j < count; j++) {
+         input[i][j] = 0;
+      }
+   }
+}
+
+static size_t factorial(size_t n)
+{
+   return (n == 0) ? 1 : n * factorial(n - 1);
+}
+
+static size_t combination(size_t n, size_t r)
+{
+   return (r > n) ? 0 : factorial(n) / (factorial(n - r) * factorial(r));
+}
+
+size_t **pascals_triangle(char *count)
```

Comment:
declared count as (char *) because , we can satisfy the test data to have a null input and also handle negative input quietly